docs/guides/how_to_guides/creating_batches/how_to_load_a_pandas_dataframe_as_a_batch.rst

```diff
@@ -145,12 +145,10 @@ This guide will help you load a Pandas DataFrame as a Batch for use in creating
 
         3. Construct a BatchRequest
 
-            We will create a ``BatchRequest`` and pass it our DataFrame via the ``batch_data`` argument.
+            We will create a ``BatchRequest`` and pass it our DataFrame via the ``batch_data`` argument. We can also optionally set ``data_asset_name``, or it will be set to  ``IN_MEMORY_DATA_ASSET`` by default.
```

Comment:
I am a little uneasy about letting `data_asset_name` be optional, unless there is a strong business requirement for this departure from the previous understanding.

Reply:
I'll close this since it's been changed in the docs to be a required parameter :)